interface/ceed-qfunction.c

```diff
@@ -573,7 +573,7 @@ int CeedQFunctionGetFlopsEstimate(CeedQFunction qf, CeedSize *flops) {
   @param[in]  f          Function pointer to evaluate action at quadrature points.
                            See \ref CeedQFunctionUser.
   @param[in]  source     Absolute path to source of QFunction, "\abs_path\file.h:function_name".
-                           For support across all backends, this source must only contain constructs supported by C99, C++11, and CUDA.
+                           This source must only contain constructs supported by the backends desired to run with (i.e. CUDA for `/gpu/cuda`, OpenCL/SYCL for `/gpu/sycl`).
```

Comment:
```suggestion
                           This source must only contain constructs supported by all targeted backends (i.e. CUDA for `/gpu/cuda`, OpenCL/SYCL for `/gpu/sycl`).
```

Maybe something like this? "desired to run with" is a bit awk to my eye

Reply:
Ah, that works better. It felt awkward when I wrote it, but couldn't figure out better phrasing.